_articles/we-dedicate-5-of-our-gross-revenue-to-environmental-causes-and-open-source-software.md

```diff
@@ -0,0 +1,86 @@
+
+We were able to grow from $400 MRR in April 2020 to more than $10,000 MRR at the start of 2021. But getting a startup to sustainability is tough and sacrifices need to be made. 
+
+Looking at my personal finances, since I joined Plausible as a co-founder in March last year, I'm down more than $27,000 in personal savings. My co-founder Uku has lost a similar level of savings since he started working on Plausible. 
+
+Just because you take a risk and start working on a startup doesn't stop the bills from coming in. We had some months without salaries and we're still not at the levels of what we used to have in the corporate world but we've made huge progress.
+
+My personal goal for this year would be to pay all my bills and other personal expenses from my Plausible salary, recover the savings lost in the first year and hopefully start saving money again. This is something that is now a realistic goal.
+
+And our stories are not uncommon. To work on something new, to try and make a difference in your own way for the things you care about, many people and teams put a lot of their time and resources. And only some can tell the story of making it work too.
+
+By us being in control of the decisions and the direction we want to take Plausible in, we don't need to cut corners or force things to achieve our personal goals. And we can still do our tiny bit to help and give back to the initiatives that we care about.
+
+And sustainability is the purpose of our donation fund. We'd like to contribute to our planet's sustainability and the sustainability of other open source projects that we rely on.
+
+## We dedicate 5% of our gross revenue to environmental causes and open source projects
+
+Because of our fortunate MRR position, we want to start contributing back. This is why we're announcing that we're taking 5% of our total revenue and giving it back.
+
+We've started this commitment on January 1st, 2021. We're allocating 5% of our gross revenue into a separate account every month. We'll distribute and donate 100% of what we have in that account at some point before the end of this year.
+
+As usual with Plausible, we will do it all with full transparency and openness to our supporters. We will pick a mix of environmental causes and open source software to donate to. And we will announce it all here.
+
+### The environmental causes 
+
+Plausible Analytics has always been a project very conscious about how we as humans impact our planet and the environment. We're 100% powered by renewable energy and we've taken many steps to be as lightweight as possible both on our own website, our stats dashboard and the script that people place on their sites.
```

Comment:
Could add a mention here to the fact that the web has a bigger carbon footprint than the whole airline industry: https://climatecare.org/infographic-the-carbon-footprint-of-the-internet/

Reply:
Pedantic but I'd specify '**our servers** are 100% powered by renewable energy'. To me it's meaningless to declare a whole company as running on renewable energy because it's an impossible goal.